From 0bb0286e7b852f3a44fad05d697958746cfaf84c Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 19 Nov 2008 14:02:21 -0800
Subject: [PATCH 15/21] Staging: comedi: comedi_pci.h: remove comedi_pci_disable_no_regions

comedi_pci_disable_no_regions is not called by any comedi code
(in or out of the kernel tree), so remove it, so no one tries to
use it in the future.

Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/comedi_pci.h |   13 -------------
 1 file changed, 13 deletions(-)

--- a/drivers/staging/comedi/drivers/comedi_pci.h
+++ b/drivers/staging/comedi/drivers/comedi_pci.h
@@ -29,19 +29,6 @@
 #include <linux/pci.h>
 
 /*
- * Called to disable PCI device if PCI device has been enabled, but
- * PCI regions have not been reserved.
- *
- * It only disables the PCI device if the kernel supports reference
- * counting of PCI enables, otherwise it might stop the device working
- * in another driver instance.
- */
-static inline void comedi_pci_disable_no_regions(struct pci_dev *pdev)
-{
-	pci_disable_device(pdev);
-}
-
-/*
  * Enable the PCI device and request the regions.
  */
 static inline int comedi_pci_enable(struct pci_dev *pdev, const char *res_name)
